### PR TITLE
Feature redux thunk

### PR DESCRIPTION
**Pull request by eldani17**
## Summary

- Thunk library was implemented.
- The format of the actions and types of the card is used.

#### Galaxy S (or similar)
Screenshot from 2021-04-03 22-30-12

## Trello Card

https://trello.com/c/nBfhO4BX/28-n-92-redux-ii-middleware

### CODE REVIEW

File: wbooks/src/app/screens/bookList/index.tsx [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 const mapStateToProps = (state: AppState) => ({
-  library: state.library
+  books: state.library.books
 });
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({
-  bookList: (books: IBook[]) => dispatch(ActionBookList(books)),
-  bookAdd: (book: IBook) => dispatch(ActionBookAdd(book))
+// dispatch: Dispatch<any>
+const mapDispatchToProps = (dispatch: any) => ({
```
Comment (mattgle) [inline wbooks/src/app/screens/bookList/index.tsx]: Change this

File: wbooks/src/app/screens/bookList/index.tsx [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+  getBookList: () => dispatch(actionCreators.getBooks())
 });
 
 interface Props {
-  library: any;
-  bookList: Function;
+  books: IBook[];
+  getBookList: any;
```
Comment (mattgle) [inline wbooks/src/app/screens/bookList/index.tsx]:
```suggestion
  getBookList:() => void;
```

**Review by mattgle — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/bookList/index.tsx (hunk 2 above)
Comment (eldani17) [reply wbooks/src/app/screens/bookList/index.tsx]: change made

**Review by mattgle — APPROVED** [review]
LGTM!

**Pull request merged**